lib/Callable/Tests.cpp

```diff
@@ -113,6 +114,21 @@ POTHOS_TEST_BLOCK("/callable/tests", test_callable_meta)
     POTHOS_TEST_EQUAL(ret2.extract<Pothos::Object>().extract<int>(), 42);
 }
 
+/***********************************************************************
+ * Using std::function
+ **********************************************************************/
+static int my_negate(const int x)
+{
+    return -x;
+}
+
+POTHOS_TEST_BLOCK("/callable/tests", test_callable_std_function)
+{
+    std::function<int(int)> my_func(my_negate);
+    Pothos::Callable my_callable(my_func);
+    POTHOS_TEST_EQUAL(my_callable.call(42), -42);
+}
+
```

Comment:
I confirmed this test on my machine.

Could you add a test with a std::function created from a std::bind? I tested this on my machine, but "it works on my machine" has bitten me before.